streaming_jupyter_integrations/magics.py

```diff
@@ -474,6 +475,59 @@ def __create_configuration_from_dict(new_values: Dict[str, Any]) -> Configuratio
 
         return configuration
 
+    @line_magic
+    def flink_show_table_tree(self, line: str) -> None:
+        return self._build_schema_tree()
+
+    def _build_schema_tree(self):
+        tree = Tree()
+        catalogs = self.st_env.execute_sql("SHOW CATALOGS").collect()
+        for catalog in catalogs:
+            tree.add_node(self._build_catalog_node(catalog))
+        return tree
+
+    def _build_catalog_node(self, catalog):
+        catalog_name = catalog[0]
+        self.st_env.execute_sql(f"USE CATALOG {catalog_name}")
+        databases = self.st_env.execute_sql(f"SHOW DATABASES").collect()
+        tree_databases = [self._build_database_node(catalog_name, database) for database in databases]
+        return Node(catalog_name, tree_databases, opened=False, icon="database")
+
+    def _build_database_node(self, catalog_name, database):
+        database_name = database[0]
+        tables = self.st_env.execute_sql(f"SHOW TABLES FROM {catalog_name}.{database_name}").collect()
+        tree_tables = [self._build_table_node(catalog_name, database_name, table) for table in tables]
+        return Node(database_name, tree_tables, opened=False, icon="database")
+
+    def _build_table_node(self, catalog_name, database_name, table):
+        table_name = table[0]
+        columns = self.st_env.execute_sql(f"DESCRIBE {catalog_name}.{database_name}.{table_name}").collect()
+        tree_columns = [self._build_column_node(column) for column in columns]
+        return Node(table_name, tree_columns, opened=False, icon="table")
+
+    def _build_column_node(self, column):
+        column_icon = "columns"
```

Comment:
It's semantically correct, but maybe we could move this variable initialization to the line 528? So instead of
```python
column_icon = "columns"
# ... blah blah...
if column_key:
  column_icon = "key"
```
it could be done in a single line:
```python
column_icon = "key" if column_key else "columns"
```
To be honest, I had to scan the whole method to ensure what are the paths that change `column_icon`. Having it in a single line would clear the things up.

Reply:
Good point. Fixed.